core/detections/best-detections.js

```diff
@@ -22,6 +22,15 @@ const Converter = require('../../common/converter')
  *         description: List of stream ids to limit results
  *         in: query
  *         type: array|string
+ *       - name: classifications
```

Comment:
```suggestion
 *       - name: classificationIds
```
some other places, we use `classifications` for classification name not ids. so better make it consistent